### DISCUSSION TITLE
IE mode support on Edge

### DISCUSSION THREAD

**Discussion by Ifera** [cypress-io/cypress#28646] [Questions and Help]
### Backstory

I am working for a client who wants their legacy app automated using Cypress. Now that app only works on Edge in IE mode and no other browser. They are in the midst of developing a new app but for now would like to automate the testing of the legacy system.

### Problem

The legacy site loads just fine on the normal Edge browser in IE mode but when the same app is accessed via Edge browser run by Cypress, it does not load since Edge does not switch into IE mode. 

### Things I have tried to get it work

I did the following steps in the Edge browser loaded by Cypress but to no avail.

- In Edge browser settings clicked Default Browser and set `Allow sites to be reloaded in Internet Explorer mode (IE mode)` to `Allow`
- Added the URL of the legacy app in there as well, as shown.

![image](https://github.com/cypress-io/cypress/assets/32965703/d1a59241-9b97-4b5e-91d3-b9a947e03526)

- Disabled `remote debugging` in Edge.

![image](https://github.com/cypress-io/cypress/assets/32965703/8271016a-c504-409f-9bed-04a9aff0bc42)

### Query

The above steps had no fruitful outcomes. Does Cypress support IE mode of Edge or no?

As seen in the screenshot below, the website loads in IE mode on Edge run manually:

![image](https://github.com/cypress-io/cypress/assets/32965703/15f56b49-5298-4c7e-b387-b459f2acb9e3)

But on cypress, this does not happen.

![image](https://github.com/cypress-io/cypress/assets/32965703/56bd4775-81dc-4358-96fb-a5e9bbf4222f)

**Comment by nehjun13** [comment]
hi @Ifera were you able to resolve this? I am also working on a similar project, but the IE mode is not getting enabled for me, despite many attempts
